### PR TITLE
Add delete to shopify_tables

### LINKED ISSUE #7820: [Shopify] Implement DELETE for the Customers table
MindsDB integrates with Shopify, so you can fetch data from your online store and use it to make relevant predictions and forecasts with MindsDB.

## Resources

Familiarize yourself with the existing Shopify integration: https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/shopify_handler

## Steps to Follow

The task is to extend [this implementation](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/shopify_handler) to include the DELETE command to delete data from the Customers table.

Please provide screenshots or a short video presenting this feature.

## :loudspeaker: Additional rewards :trophy:

Each integration extension brings 15+ points for SWAG and entry into the draw for a :point_down: :
-  :computer:  [Razer Blade 16 Laptop](https://www.razer.com/gaming-laptops/Razer-Blade-16/RZ09-0483SEJ3-R3U1)
- :shirt: :bear:  [MindsDB Swag](https://store.covver.io/mindsdb/collections/swag-collection)

:information_source: For more info check out https://mindsdb.com/hacktoberfest/ :point_left: 

**Please note that this issue is for first-time contributors to MindsDB**

### PR DESCRIPTION
**Pull request by Smartmind12**
## Description

The `delete_customers` function takes a list of customer IDs as input and then uses the Shopify API to delete each customer

Fixes #7820 

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [X] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] 📄 This change requires a documentation update

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [ ] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [X] My code follows the style guidelines(PEP 8) of MindsDB.
- [X] I have appropriately commented on my code, especially in complex areas.
- [ ] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Comment by martyna-mindsdb** [comment]
@Smartmind12 
Please provide screenshots or a short video presenting this feature, as written in the issue description.

**Comment by Smartmind12** [comment]
> @Smartmind12 Please provide screenshots or a short video presenting this feature, as written in the issue description.

Screenshot 2023-10-16 at 10 37 38 PM
Screenshot 2023-10-16 at 10 44 40 PM

File: mindsdb/integrations/handlers/shopify_handler/shopify_tables.py [hunk 1]
```diff
@@ -317,6 +317,15 @@ def update_customers(self, customer_ids: List[int], values_to_update: List[Dict[
             customer.save()
             logger.info(f'Customer {customer_id} updated')
 
+    def __init__(self, handler: ShopifyHandler):
```
Comment (martyna-mindsdb) [inline mindsdb/integrations/handlers/shopify_handler/shopify_tables.py:320]: This `__init__` function is not necessary here. Please see this implementation for an example: https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/shopify_handler/shopify_tables.py#L86

Comment (Smartmind12) [reply mindsdb/integrations/handlers/shopify_handler/shopify_tables.py]: Yess i have considered it and i have added some lines of code to make it work hence added a commit to solve both issues.

Comment (martyna-mindsdb) [reply mindsdb/integrations/handlers/shopify_handler/shopify_tables.py]: Thanks. Did you test it after making these changes?

**Comment by martyna-mindsdb** [comment]
@Smartmind12 
Thanks. Did you test it after making these changes?

File: mindsdb/integrations/handlers/shopify_handler/shopify_tables.py (hunk 1 above)
Comment (Smartmind12) [reply mindsdb/integrations/handlers/shopify_handler/shopify_tables.py]: Yes i had made changes in the gitpod directory and not in github files so i have done that now hence the screenshots are of earlier gitpod activation.

**Comment by martyna-mindsdb** [comment]
@Smartmind12 
Please add a sample DELETE statement to the README file.

**Comment by Smartmind12** [comment]
> @Smartmind12 Please add a sample DELETE statement to the README file.

Yupp done!

**Comment by martyna-mindsdb** [comment]
@Smartmind12 
Please provide a short video that presents the following:
- SELECT FROM shopify_datasource.customers table
- DELETE FROM shopify_datasource.customers table
- again, SELECT FROM shopify_datasource.customers table to show the deleted row is not there

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]
@Smartmind12 Can you please solve the conflicts?

**Comment by ZoranPandovski** [comment]
@Smartmind12 Can you please solve the conflicts so we can merge this?

**Comment by Smartmind12** [comment]
> @Smartmind12 Can you please solve the conflicts so we can merge this?

Hey @ZoranPandovski I have done the needful!

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**